operations/firmware/Firmware Upgrade using SPP on HPE ProLiant Servers.md

```diff
@@ -0,0 +1,105 @@
+# Firmware Upgrade using SPP on HPE ProLiant Servers
+#### Link to download SPP Image : 
+https://techlibrary.hpe.com/us/en/enterprise/servers/products/service_pack/spp/index.aspx
+
+* Naming convention for example : 2023.04.00.00 that's the ISO image released in 4th Month of 2023
+
+* Download the latest version or the version you needed.
+
+* We can either deploy individual firmware component or deploy SPP ISO image as a whole
+
+### How to deploy individual firmware component?
+
+####  Get the individual firmware component from SPP ISO image:
+
+* If we mount the SPP ISO image or extract the iso image, we can see there is “contents.html” file present.
+* For example, if we want to flash the BIOS/ROM for DL360 Gen11 server then find for DL360 in “Description” column, and you see the relevant component to flash through ILO. Note the extension should be fwpkg if we are flashing through ILO.

```

Comment:
```suggestion
* Get the location of the file/package in the extracted/mounted ISO image from the "contents.html" file.  Usually it will be in the folder "Packages".
* For example, to flash the BIOS/ROM for DL360 Gen11 server, find DL360 in “Description” column, and then find relevant component to flash. Note the extension should be `.fwpkg` if we are flashing through ILO.

```